### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Created Product listing page which specifies sales, fresh arrivals and filters.

### CODE REVIEW

File: Pages/Product-Listing-Page/index.html [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+                <div class="card-txt-container">
+                    <div>
+                        <h6 class="badge-upper">Sale 40% off</h6>
+                    </div>
+                    <img alt="card-img-headphone" class="basic-img" src="/assets/dog-bed.png">
+                    <div class="txt-margin">
+                        <div>
+                            <h2 class="card-heading">Dog Bed</h2>
+                            <h3 class="card-subheading">Rs. 9,600 <span class="card-txt-red"
+                                    style="font-size: 12px; text-decoration:line-through">Rs. 15,999 </span><span
```
Comment (pradipgharge) [inline Pages/Product-Listing-Page/index.html]: Avoid Inline styling here.

File: Pages/Product-Listing-Page/index.html [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        <h4>Created by Vanshita Moolchandani</h4>
+        <h2 class="heading-alignc"></h2>
+        <ul class="list">
+            <li class="list-type-inline">
+                <a class="link" href="https://www.linkedin.com/in/vanshita-moolchandani/">
+                    <i class="fab fa-linkedin social"></i>
+                </a>
+            </li>
+            <li class="list-type-inline">
+                <a class="link" href="">
```
Comment (pradipgharge) [inline Pages/Product-Listing-Page/index.html:231]: The Twitter link is missing.

**Review by pradipgharge — COMMENTED** [review]
Looks good, except remove the inline styling in cards.

File: Pages/Product-Listing-Page/index.html (hunk 1 above)
Comment (vanshitamoolchandani) [reply Pages/Product-Listing-Page/index.html]: Yeah Thank you for pointing it out

**Comment by prankurpandeyy** [comment]
Looks good to me can you align cart icon in button that would be more relatable, rest is good .

**Review by prankurpandeyy — APPROVED** [review]

File: Pages/Product-Listing-Page/index.html (hunk 2 above)
Comment (vanshitamoolchandani) [reply Pages/Product-Listing-Page/index.html:231]: Added

**Pull request merged**